sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpException.java

```diff
@@ -71,6 +91,13 @@ public String getPayload() {
         return payload;
     }
 
+    /**
+     * Get the HTTP headers (response body).
+     *
+     * @return HTTP headers.
+     */
+    public Map<String, String> getHeaders() { return headers; }
```

Comment:
Please properly format the changes (not only this one)

Reply:
cmd/ctrl-shift-L is the default formatting keybind which should get you closer.  It may not fix issues like this though